receiver/gitproviderreceiver/README.md

```diff
@@ -0,0 +1,32 @@
+# Git Provider Receiver
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: metrics   |
+| Distributions | [] |
+| Issues        | ![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Areceiver%2Fgitprovider%20&label=open&color=orange&logo=opentelemetry) ![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Areceiver%2Fgitprovider%20&label=closed&color=blue&logo=opentelemetry) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+<!-- end autogenerated section --> 
+
+The Git Provider receiver scrapes data from Git vendors.
+
+## Getting Started
+
+The collection interval is common to all scrapers and is set to 30 seconds by default.
+
+```yaml
+gitprovider:
```

Comment:
Please include the typical options people would use to get started with this, even if it's not implemented yet (it's marked as "development" after all). If GitHub is what we provide today, perhaps give a full example of GitHub, including how to set the token (might involve other extensions for that?)